fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeMetadata.java

```diff
@@ -71,13 +70,13 @@ public class DeltaLakeMetadata implements ConnectorMetadata {
     private final String catalogName;
     private final DeltaMetastoreOperations deltaOps;
     private final HdfsEnvironment hdfsEnvironment;
-    private final Optional<DeltaLakeCacheUpdateProcessor> cacheUpdateProcessor;
-    private final Map<PredicateSearchKey, List<Row>> splitTasks = new ConcurrentHashMap<>();
+    private final DeltaLakeCacheUpdateProcessor cacheUpdateProcessor;
```

Comment:
Can you elaborate a little bit why change this

Reply:
This does not comply with Java programming specifications. Optional is generally used for function return values. Java does not recommend using it for parameters or data members.